logic/suggestions/handToss.go

```diff
@@ -0,0 +1,72 @@
+package suggestions
```

Comment:
:book: what is the naming convention for files in go? This one is in camelCase, but the one above is in snake_case. Is one more idiomatic than the other?

Reply:
yea - camel case is supposed to be the "way". I'm a bad go dev occasionally:#